index.html

```diff
@@ -78,6 +64,76 @@ <h1 class="menu-title">Menu items</h1>
     <div class="container"></div>
   </main>
 
+  <!-- MODALS -->
+  <div class="modals">
+    <a href="#" id="modal-button" class="button add">Add</a>
+  </div>
+  <div id="addItemModal" class="modal">
+    <!-- Modal content -->
+    <div class="modal-content slideDown">
+      <div class="modal-header">
+        <span class="close closeAddModal" id="closeModal">&times;</span>
+        <h2>Add item</h2>
+      </div>
+      <div class="modal-body">
+        <form action="" class="modal-form">
+          <div class="form-row">
+            <label for="">Title</label>
+            <input type="text" id="title"></input>
+          </div>
+          <div class="form-row">
+            <label for="">Description</label>
+            <input type="text" id="description">
+          </div>
+          <div class="form-row">
+            <label for="">Price</label>
+            <input type="number" id="price" step=".01">
+          </div>
+          <div class="form-row">
+            <label for="">Image URL</label>
+            <input type="text" id="imageUrl">
+          </div>
+      </div>
+      <div class="modal-footer">
+        <input type="submit" class="button add-product" value="Add">
```

Comment:
Consider using `button` with a `submit` type. You'd probably need less styling to achieve the needed looks in this case.